docs/can-guides/experiment/crud-beginner/crud-beginner.md

```diff
@@ -136,15 +139,15 @@ Using this component will insert the following into the page:
 
 The next two sections will explain these lines.
 
-### Defining properties on the ViewModel
+### Defining properties
 
-Every time a component’s custom element is used, a new instance of the component’s `ViewModel` is created.
+Every custom element instance has the properties listed in `define` available to it.
```

Comment:
I think this is a little confusing:

> available to it

Maybe

> Each time a custom element is created, each property listed in `define` will be added to it.

or

> Each time a custom element is created, each property listed in `define` will be defined on the instance.

Reply:
+1 for the second phrasing